src/html/gnc-html-webkit.c

```diff
@@ -108,7 +108,7 @@ static void impl_webkit_show_url( GncHtml* self, URLType type,
                                   const gchar* location, const gchar* label,
                                   gboolean new_window_hint );
 static void impl_webkit_show_data( GncHtml* self, const gchar* data, int datalen );
-static void impl_webkit_reload( GncHtml* self );
+static void impl_webkit_reload( GncHtml* self, gboolean view );
```

Comment:
"view" as variable name here is not really describing very well what the boolean is doing. Better would have been for example "force_reconstruct" or  "force_reparse" or something like that although those names invert the meaning of the boolean. But with a name like that it's easier to guess what it does.